samples/v2/Makefile

```diff
@@ -16,9 +16,9 @@
 .PHONY: all
 all:
 	@for f in $$(find . -name "*.py"); do \
-		compiled="$${f%.*}_pipeline.json"; \
+		compiled="$${f%.*}_pipeline.yaml"; \
```

Comment:
I think you'll need to update the `clean` Make target as well since it looks for all JSON files still.